apple/OmnivoreKit/Sources/Views/Resources/en.lproj/Localizable.strings

```diff
@@ -10,8 +10,8 @@
 
 "labels.generic" = "Labels";
 "emails.generic" = "Emails";
-"subscriptions.generic" = "Text to Speech";
```

Comment:
This is what happens when I stop drinking real coffee 😬

Reply:
Hah, amazingly a user reported within an hour of testflight being published